SU2_CFD/src/fluid/CFluidFlamelet.cpp

```diff
@@ -72,7 +72,7 @@ CFluidFlamelet::CFluidFlamelet(CConfig* config, su2double value_pressure_operati
 
   table_source_names.resize(n_table_sources);
   table_sources.resize(n_table_sources);
-  table_source_names[I_SRC_TOT_PROGVAR] = "ProdRateTot_PV";
+  table_source_names[I_SRC_TOT_PROGVAR] = "ProdRateTot-PV";
```

Comment:
I would like to call it ProdRateTot_PV in the latest dragon format. The logic is <quantity>_<specifier> and inside <specifier> we can use "-" to separate words. Also, e.g.:  ProdRatePos_Y-CO. I know this is an inconvenience
for existing dragon tables with the other nomenclature. My proposal is to rename this in your existing tables now for the last time and in the future we will handle changes to the hard coded variable names through version numbers of the dragon file. Hope that works for everybody. Please let me know.